Sources/RichTextKit/Actions/RichTextAction.swift

```diff
@@ -149,11 +132,15 @@ public extension RichTextAction {
         case .stepSuperscript(let steps): .actionStepSuperscript(steps)
         case .toggleStyle(let style): style.titleKey
         case .undoLatestChange: .actionUndoLatestChange
-        case .setBackgroundColor: .backgroundColor
-        case .setForegroundColor: .foregroundColor
-        case .setUnderlineColor: .underlineColor
-        case .setStrikethroughColor: .strikethroughColor
-        case .setStrokeColor: .strokeColor
+        case .setColor(_, let richTextColor):
+            switch richTextColor {
+            case .foreground: .foregroundColor
+            case .background: .backgroundColor
+            case .strikethrough: .strikethroughColor
+            case .stroke: .strokeColor
+            case .underline: .underlineColor
+            case .undefined: .strokeColor
```

Comment:
Is this correct for the undefined?

Reply:
You need to return something and this feels a lot better than exception or cascade of optionals. I would personally focus after 1.0 to get away from `undefined`...